documentation/help_main.txt

```diff
@@ -1047,6 +1047,7 @@ information, as follows:
 
 :Upload type: Choose whether the sample is a file or metadata
 :Filename: The name of the file encountered
+:FIlepath: The path of the file encountered

```

Comment:
Looks like the `I` needs to be lowercase. I don't see this error reflected in the `help_main.html` file which makes me think you did not use `rst2html.py` to re-generate the `help_main.html` file. You should never edit the html file by hand. Always re-generate it off the txt file.